[개선] 접수증 양식 변경

jyj1289 opened the pull request:
- 접수증에 대상구분을 추가하고 전형구분을 메인카테고리를 하도록 변경했어요.

## 🎫 관련 이슈
[//]: # (다음 키워드를 사용하면 해당 PR을 머지할 때 자동으로 이슈를 닫을 수 있습니다.)
[//]: # (keyword: close|closes|closed|resolve|resolves|resolved|fix|fixes|fixed)
[//]: # (예시: close #1)

close #152 

<br>

## 📄 개요
[//]: # (작업 내용을 간단히 요약해서 적습니다.)
[//]: # (예시: 유저 회원가입 기능을 만들었습니다.)

> 접수증 양식을 변경했습니다.

<br>

## 🔨 작업 내용
[//]: # (작업 내용을 자세하게 적습니다.)
[//]: # (붙임표 "-" 을 사용해서 목록을 만듭니다.)
[//]: # (예시: 유저 회원가입 API를 만들었습니다.)

- 접수증에 대상구분 값을 추가했습니다.


<br>

## 🏁 확인 사항
[//]: # (PR을 보내기 전 다음 사항을 확인해주세요.)
[//]: # (해당 사항을 모두 이행해야 머지할 수 있습니다.)
[//]: # (- [x] 를 사용해서 완료로 표시할 수 있습니다.)

- [x] 테스트를 완료했나요?
- [x] API 문서를 작성했나요?
- [x] 코드 컨벤션을 준수했나요?
- [x] 불필요한 로그, 주석, import 등을 삭제했나요?

<br>

## 🙋🏻 덧붙일 말
[//]: # (다음 사항이 있다면 적어주세요.)
[//]: # (PR에 대한 추가 설명)
[//]: # (중점적으로 리뷰받고 싶은 부분)
[//]: # (기타 등등)

jyj1289 committed:
perf(#152): 접수증 양식 변경
- 접수증에 대상구분을 추가하고 전형구분을 메인카테고리를 하도록 변경했어요.

cabbage16 commented:
P4) 우연히 발견한건데 혹시 메서드에 오타 수정 가능한가요?
![image](https://github.com/user-attachments/assets/be8569df-dafe-4b44-b828-0f5e6ac6ed63)

cabbage16 reviewed (changes requested):
P2)
다른 전형은 다 되는데
![image](https://github.com/user-attachments/assets/f2ea4d4c-5bc1-441e-b38f-1c45b4d6f050)

국가보훈대상자 중 교육지원대상자녀는 이름이 너무 길어서 다른칸이 이상하게 나오네요
![image](https://github.com/user-attachments/assets/0630d0a9-8c9a-44d2-bd60-8e142fc9362f)

jyj1289 committed:
fix(#152): 오타수정
- FormController 에 generateProfOfApplication 메서드 오타를 generateProofOfApplication 메서드로 저장했어요.

jyj1289 committed:
fix(#152): 국가보훈대상자 중 교육지원대상자녀 전형시 값 변경
- 국가보훈대상자 중 교육지원대상자녀 전형일 때 칸이 깨지는 것을 방지하기 위해 '국가보훈대상자'로 값을 바꾸었어요.
